### PR TITLE
Add jll for Julia 1.3 and higher

### PR DESCRIPTION
**Pull request by odow**
x-ref: https://github.com/JuliaPackaging/Yggdrasil/issues/509#issuecomment-605516668

### CODE REVIEW

**Comment by odow** [comment]
Any ideas? BinaryProvider installs 2.0.5, whereas ECOS_jll installs 2.0.7.

For some reason, 2.0.7 no longer detects infeasibility in one of the MPB tests, and it hits an iteration limit:
https://ci.appveyor.com/project/JuliaOpt/ecos-jl/builds/31805100/job/gfs908v4n55d4y78#L384

**Comment by odow** [comment]
Nothing in the diff suggests a reason for the test failing: https://github.com/embotech/ecos/compare/v2.0.5...2.0.7

**Comment by amontoison** [comment]
I can build ECOS `2.0.5` with Yggdrasil if you want to keep the same version with Julia < 1.3.

**Comment by odow** [comment]
@amontoison yes please. That's a good short-term fix. 

In the long-term, it would be nice to understand why things changed.

File: Project.toml [hunk 1]
```diff
@@ -13,6 +14,7 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
 BinaryProvider = "0.5"
+ECOS_jll = "2.0"
```
[the remark this answers is not in the document]
Comment (giordano) [reply Project.toml:17]: More specifically
```suggestion
ECOS_jll = "=2.0.5"
```

**Comment by ViralBShah** [comment]
It's still picking 2.0.7.

**Comment by amontoison** [comment]
https://github.com/JuliaRegistries/General/pull/11905 was not merged.

**Comment by ViralBShah** [comment]
I merged it. Automerge is not working it seems. @giordano I hope its ok to merge manually.

Close/open this PR again?

**Comment by ViralBShah** [comment]
Huh - not sure why it is not picking up 2.0.5 on travis.

**Comment by odow** [comment]
I kicked it off again. Travis must have got confused during the restart somehow. The other builds passed.

**Comment by giordano** [comment]
Am I missing something in https://travis-ci.org/github/JuliaOpt/ECOS.jl/builds/669469878?  I read
```
  [c2c64177] + ECOS_jll v2.0.5+0
```
in the log

**Pull request merged**